client/hooks/packs.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { queryTrpc } from "../trpc";
+import { tr } from "date-fns/locale";
+
+export const usePacks =  (queryString, ownerId, feedType, selectedTypes) => {
+    console.log(feedType,'feed type', ownerId,queryString, 'query string',selectedTypes)
+    let  data = [];
+    let isLoading = true;
+    switch (feedType) {
+        case 'public':
+            const publicPacks = queryTrpc.getPublicPacks.useQuery({ queryBy:queryString ? queryString:  'Favorite' }, {
```

Comment:
We should at the very least be destructuring isLoading and error from the react query responses. To make it more like the redux store